stretch_gazebo/nodes/keyboard_teleop_gazebo

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import math
+import keyboard as kb
+
+import rospy
+from sensor_msgs.msg import JointState
+from control_msgs.msg import FollowJointTrajectoryGoal, FollowJointTrajectoryAction
+from trajectory_msgs.msg import JointTrajectoryPoint
+from geometry_msgs.msg import Twist
+import actionlib
+
+
+# NOTE: FUNMAP services and capabilities won't work with Stretch Gazebo simulation
+
+class GetKeyboardCommands:
+    def __init__(self):
+        self.step_size = 'medium'
+        self.rad_per_deg = math.pi / 180.0
+        self.small_deg = 3.0
+        self.small_rad = self.rad_per_deg * self.small_deg
+        self.small_translate = 0.005  # 0.02
+        self.medium_deg = 6.0
+        self.medium_rad = self.rad_per_deg * self.medium_deg
+        self.medium_translate = 0.04
+        self.big_deg = 12.0
+        self.big_rad = self.rad_per_deg * self.big_deg
+        self.big_translate = 0.06
+        self.mode = 'position'  # 'manipulation' #'navigation'
```

Comment:
Since Gazebo doesn't have a position mode controller, let's disable the mode here.

Reply:
Position mode support is removed throughout the code.